XSLT/knoxcountydctomods_newsets.xsl

```diff
@@ -0,0 +1,169 @@
+        
+        <!-- match the document root and return a MODS record -->
+                <mods xmlns="http://www.loc.gov/mods/v3" version="3.5"
+                    xmlns:xlink="http://www.w3.org/1999/xlink"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+                    
+                    <!-- title -->
+                    <xsl:apply-templates select="dc:title"/>    
+                    
+                    <!-- identifier -->
+                    <xsl:apply-templates select="dc:identifier"/>
+                    
+                    <!-- description (minus transcriptions) -->
+                    <xsl:apply-templates select="dc:description[1]"/>
+                    
+                    <!-- creator -->
+                    <xsl:apply-templates select="dc:creator"/>
+                    
+                    <!-- contributor -->
+                    <xsl:apply-templates select="dc:contributor"/>
+                    
+                    <!-- originInfo> -->
+                    <xsl:apply-templates select="dc:date"/>
+                    
+                    <!-- subject(s) -->
+                    <xsl:apply-templates select="dc:subject"/>
+                    
+                    <!-- form -->
+                    <xsl:apply-templates select="dc:type"/>
+                    
+                    <!-- recordContentSource -->
+                    <recordInfo>
+                        <recordContentSource>Knox County Public Library</recordContentSource>
+                    </recordInfo>
+                    
+                    <!-- accessCondition -->
+                    <accessCondition type="use and reproduction" xlink:href="http://rightsstatements.org/vocab/CNE/1.0/">Copyright Not Evaluated</accessCondition>
+                    
+                </mods>
+    </xsl:template>
+    
+    <!-- title -->
+    <xsl:template match="dc:title">
+        <titleInfo>
+            <title><xsl:value-of select="normalize-space(.)"/></title>
+        </titleInfo>
+    </xsl:template>
+    
+    <!-- identifiers -->
+    <xsl:template match='dc:identifier'>
+        <xsl:choose>
+            <xsl:when test="starts-with(., 'http://')">
+                <xsl:variable name="identifier-preview-url" select="replace(., '/cdm/ref', '/utils/getthumbnail')"/>
+                <location>
+                    <url usage="primary" access="object in context"><xsl:apply-templates/></url>
+                    <url access="preview"><xsl:value-of select="$identifier-preview-url"/></url>
+                </location>
+            </xsl:when>
+            <xsl:otherwise>
+                <identifier><xsl:value-of select="normalize-space(.)"/></identifier>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+    
+    <!-- abstract -->
+    <xsl:template match="dc:description">
```

Comment:
Normally, I'd disambiguate my xsl:template match to match my xsl:apply-templates statement. This is important if you had multiple rules, but doesn't specifically matter here.

I checked if we had a contributing rule about this here, but I see nothing.  Therefore, feel free to change or leave as is.

```
<xsl:template match="dc:description[1]">
```

Reply:
This makes sense to change so that both are the same - thanks. I've committed the change.